content/resources/user/submit-data/sower.md

```diff
@@ -5,24 +5,24 @@ linktitle: /resources/user/submit-data
+Data files stored on cloud services, such as Amazon Web Services (AWS) or Google Cloud Storage (GCS), can be linked to a Gen3 data commons via normal submission and indexing. Below are listed guidelines and considerations regarding the specific processes of submission and indexing.
 
-1. Consider required authorization controls for the data and how/where Gen3 will synchronize user authorization information. This information is important to communicate with CTDS/commons operators. Depending on your data access and authorization needs, there are some requirements and caveats to data organization in the buckets.
+1. Consider required authorization controls for the data and how/where Gen3 will synchronize user authorization information. This information is important to communicate with CTDS/commons operators. Depending on your data access and authorization requirements, there are some requirements and caveats to data organization in the buckets.
 
-2. In order for Gen3 to make file objects available to researchers, globally unique identifiers (GUIDs) are minted per object and stored in Gen3. That process of minting GUIDs associated with specific file objects in buckets is generally achieved by providing Gen3 with a bucket manifest which contains the minimal file-level metadata required to mint a GUID. A bucket manifest as a TSV file **has** to contain the following properties at the minimum: <ins>md5</ins> checksum, file <ins>size</ins> in bytes, and the full bucket <ins>urls</ins>. There are authorization columns "authz" and "acl" that can also be populated, though knowing what to populate these with will require communications with Gen3/commons operators.
+2. In order for Gen3 to make file objects available to researchers, globally unique identifiers (GUIDs) are minted per object and stored in Gen3. The process of minting GUIDs associated with specific file objects in buckets is achieved with a bucket manifest which contains minimum file-level metadata. A bucket manifest is a tab seperated variable (TSV) file containing the following properties:
 
-    > __Note:__ Gen3's automation and deployment code contains scripts for [GCS](https://github.com/uc-cdis/cloud-automation/blob/master/doc/gcp-bucket-manifest.md) and [AWS](https://github.com/uc-cdis/cloud-automation/blob/master/doc/bucket-manifest.md) that can generate an object manifest of a bucket. Keep in mind that using these scripts requires using Gen3's full deployment code and, depending on the amount of data, calculating checksums for files can be costly and take time.
+| md5_checksum | file_size | bucket_urls | acl | authz |
```

Comment:
We are strongly recommending the inclusion of file_name in the manifest as well.